app/services/update_user.rb

```diff
@@ -33,6 +28,34 @@ def update_sam
     reckoner.delay.set!
   end
 
+  def update_auctions
+    if added_payment_information?
+      AuctionQuery.new.accepted_with_bid_from_user(user.id).each do |auction|
+        if auction.accepted_at.nil? && winning_bidder_for(auction) == user
```

Comment:
Should we take this condition and make it a method? `user_payment_info_needed_for_auction` or such?